pkg/wait/test_wait_helper.go

```diff
@@ -42,7 +42,7 @@ func NewFakeWatch(events []watch.Event) *FakeWatch {
 	}
 }
 
-// Stop the watch challend
+// Stop the watch channel called
```

Comment:
I have absolutely no idea what the right item is here.
Things I considered:
`channel`, `called`, `channel called`, `called count`

This was chosen after the equivalent of a coin toss. I asked some others for opinions and ...

Reply:
`Stop the watch channel`